npe2/_inspection/_full_install.py

```diff
@@ -0,0 +1,117 @@
+"""This module is mostly superceded by the static NPE1ModuleVisitor pattern.
```

Comment:
this module is just moved from (and split out from) npe2/_fetch.  no new code in here